database/seeders/json/roles.json

```diff
@@ -0,0 +1,234 @@
+[
+    {
+        "name": "doctor",
+        "title": "Doctor",
+        "permissions": [
+            {
+                "ability": "view-customer",
+                "model": "Crater\\Models\\Customer",
+                "id": 1
```

Comment:
Would it be possible to remove `id` attribute from here and may be we use the index while populating data using the seeder ?

Reply:
I don't think it is possible since the ids are not directly indexes. The permissions have different ids and not really they are not linear.